docs/reference/feature-gates.md

```diff
@@ -60,7 +60,7 @@ different Kubernetes components.
 | `LocalStorageCapacityIsolation` | `false` | Alpha | 1.7 | 1.9 |
 | `LocalStorageCapacityIsolation` | `true` | Beta| 1.10 | |
 | `MountContainers` | `false` | Alpha | 1.9 | |
-| `MountPropagation` | `false` | Alpha | 1.8 | |
```

Comment:
We keep tracking the status of each feature gates so you may want to revise this line AND add the following line:

```
| `MountPropagation` | `false` | Alpha | 1.8 | 1.9  |
```

Reply:
Fixed